package.json

```diff
@@ -96,7 +96,7 @@
     "watch-js-docs": "nodemon --watch site/assets/js/ --ext js --exec \"npm run js-lint\"",
     "storybook": "npm run storybook-generate && storybook dev -p 6006",
     "storybook-generate": "npm run docs && node stories/create-stories-from-doc.js",
-    "storybook-build": "npm run storybook-generate && storybook build -o ./_site/storybook",
+    "storybook-build": "npm run storybook-generate && storybook build -o ./_site/ouds-web/storybook",
     "storybook-build-netlify": "npm run storybook-generate && storybook build -o ./site/storybook"
```

Comment:
Do we keep only one url for storybook on prod? I'd say no, but maybe you have something in mind?

Reply:
What do you mean by that?
For now, on Boosted, you only have boosted.orange.com/storybook (latest version) since it's only used by Zeroheight